extensions/amp-story-interactive/0.1/amp-story-interactive-img.css

```diff
@@ -91,3 +95,52 @@
   color: white !important;
   font-weight: 700 !important;
 }
+
+.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection)
+  .i-amphtml-story-interactive-img-option-img {
+  animation: i-amphtml-story-interactive-img-option-img-landing-animation
+    var(--i-amphtml-interactive-animation-time) !important;
+  animation-delay: var(
+    --i-amphtml-interactive-landing-animation-delay-offset-el1
+  ) !important;
+}
+
+.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection)
+  .i-amphtml-story-interactive-img-option-img::before {
+  display: block !important;
+  background-color: var(--i-amphtml-interactive-theme-shading-flash) !important;
+  opacity: 0 !important;
+  animation: i-amphtml-interactive-animation-landing-animation-flash-background
+    var(--i-amphtml-interactive-animation-time) !important;
+  animation-delay: inherit !important;
+}
+
+.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection)
+  .i-amphtml-story-interactive-img-option:nth-of-type(2)
+  .i-amphtml-story-interactive-img-option-img {
+  animation-delay: var(
+    --i-amphtml-interactive-landing-animation-delay-offset-el2
+  ) !important;
+}
+
+.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection)
+  .i-amphtml-story-interactive-img-option:nth-of-type(3)
+  .i-amphtml-story-interactive-img-option-img {
+  animation-delay: var(
+    --i-amphtml-interactive-landing-animation-delay-offset-el3
+  ) !important;
+}
+
+.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection)
+  .i-amphtml-story-interactive-img-option:nth-of-type(4)
+  .i-amphtml-story-interactive-img-option-img {
+  animation-delay: var(
+    --i-amphtml-interactive-landing-animation-delay-offset-el4
+  ) !important;
+}
```

Comment:
The animation delay could use a simpler selector like `.i-amphtml-story-interactive-img-option:nth-of-type(4) .i-amphtml-story-interactive-img-option-img` since the other part of the selector does not affect this (when the component is not active, the animation will just not play).